### PR TITLE
Store instructions, rs5, workflow, install method

### PR DESCRIPTION
**Pull request by evmill**
Removed old instructions on tap to place apps from pre rs5
Clarified not all apps need to be purchased.
Added note you can use MSA while using another account.
Got rid of a BLOOM reference corrected to Start Gesture 
Added My Library reference
@scooley

### CODE REVIEW

File: devices/hololens/holographic-store-apps.md [hunk 1]
```diff
@@ -33,20 +33,26 @@ Open the Microsoft Store from the **Start** menu. Then browse for apps and games
...
+>[!NOTE]
+>The account you can use on Microsoft Store does not have to be the same as the account you are signed in with. If you are using a Work or School account on your HoloLens then you'll need to sign in with your personal account in the Store App to make a purchase.
 
-1. To open the [**Start** menu](holographic-home.md), perform a [bloom](hololens1-basic-usage.md) gesture or tap your wrist.
-2. Select the Store app and then tap to place this tile into your world.
-3. Once the Store app opens, use the search bar to look for any desired application.
-4. Select **Get** or **Install** on the application's page (a purchase may be required).
+To set up a payment method, go to [account.microsoft.com](https://account.microsoft.com/) and select **Payment & billing** > **Payment options** > **Add a payment option**.
+
+1. To open the [**Start** menu](holographic-home.md), perform a the [Start gesture](https://docs.microsoft.com/en-us/hololens/hololens2-basic-usage#start-gesture) or [bloom](hololens1-basic-usage.md) gesture on HoloLens 1.
```
Comment (scooley) [inline devices/hololens/holographic-store-apps.md]: @evmill - can you remove the "en-us" from this URL?

Comment (evmill) [reply devices/hololens/holographic-store-apps.md]: @scooley Removed. I'll try to get in the habit of removing them going forward.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: devices/hololens/holographic-store-apps.md [hunk 2]
```diff
@@ -33,20 +33,26 @@ Open the Microsoft Store from the **Start** menu. Then browse for apps and games
 
 ## Install apps
 
-To download apps, you'll need to be signed in with a Microsoft account. To buy them, you'll need a payment method associated with the Microsoft account you use on your HoloLens. To set up a payment method, go to [account.microsoft.com](https://account.microsoft.com/) and select **Payment & billing** > **Payment options** > **Add a payment option**.
+To download apps, you'll need to be signed in with a Microsoft account. Some apps are free and can be downloaded right away. Apps that require a purchase require you to be signed into the Store with your Microsoft account and have a valid payment method.
```
Comment (JohanFreelancer9) [inline devices/hololens/holographic-store-apps.md]:
```suggestion
To download apps, you'll need to be signed in with a Microsoft account. Some apps are free and can be downloaded right away. Apps that require a purchase require you to be signed into the Store with your Microsoft account and have a valid payment method.
```

File: devices/hololens/holographic-store-apps.md [hunk 3]
```diff
@@ -33,20 +33,26 @@ Open the Microsoft Store from the **Start** menu. Then browse for apps and games
...
+>[!NOTE]
```
Comment (JohanFreelancer9) [inline devices/hololens/holographic-store-apps.md]:
```suggestion
> [!NOTE]
```

File: devices/hololens/holographic-store-apps.md [hunk 4]
```diff
@@ -33,20 +33,26 @@ Open the Microsoft Store from the **Start** menu. Then browse for apps and games
...
+>The account you can use on Microsoft Store does not have to be the same as the account you are signed in with. If you are using a Work or School account on your HoloLens then you'll need to sign in with your personal account in the Store App to make a purchase.
 
-1. To open the [**Start** menu](holographic-home.md), perform a [bloom](hololens1-basic-usage.md) gesture or tap your wrist.
-2. Select the Store app and then tap to place this tile into your world.
-3. Once the Store app opens, use the search bar to look for any desired application.
-4. Select **Get** or **Install** on the application's page (a purchase may be required).
+To set up a payment method, go to [account.microsoft.com](https://account.microsoft.com/) and select **Payment & billing** > **Payment options** > **Add a payment option**.
+
+1. To open the [**Start** menu](holographic-home.md), perform a the [Start gesture](https://docs.microsoft.com/hololens/hololens2-basic-usage#start-gesture) or [bloom](hololens1-basic-usage.md) gesture on HoloLens 1.
```
Comment (JohanFreelancer9) [inline devices/hololens/holographic-store-apps.md]:
```suggestion
1. To open the [**Start** menu](holographic-home.md), perform a [Start gesture](https://docs.microsoft.com/hololens/hololens2-basic-usage#start-gesture) or [bloom](hololens1-basic-usage.md) gesture on HoloLens 1.
```

File: devices/hololens/holographic-store-apps.md [hunk 5]
```diff
@@ -33,20 +33,26 @@ Open the Microsoft Store from the **Start** menu. Then browse for apps and games
...
+1. Select the Store app. Once the Store app opens:
+  1. Use the search bar to look for any desired application.
```
Comment (JohanFreelancer9) [inline devices/hololens/holographic-store-apps.md]:
```suggestion
  1. Use the search bar to look for any desired applications. 
```

File: devices/hololens/holographic-store-apps.md [hunk 6]
```diff
@@ -33,20 +33,26 @@ Open the Microsoft Store from the **Start** menu. Then browse for apps and games
...
-1. To open the [**Start** menu](holographic-home.md), perform a [bloom](hololens1-basic-usage.md) gesture or tap your wrist.
-2. Select the Store app and then tap to place this tile into your world.
-3. Once the Store app opens, use the search bar to look for any desired application.
-4. Select **Get** or **Install** on the application's page (a purchase may be required).
+To set up a payment method, go to [account.microsoft.com](https://account.microsoft.com/) and select **Payment & billing** > **Payment options** > **Add a payment option**.
+
+1. To open the [**Start** menu](holographic-home.md), perform a the [Start gesture](https://docs.microsoft.com/hololens/hololens2-basic-usage#start-gesture) or [bloom](hololens1-basic-usage.md) gesture on HoloLens 1.
+1. Select the Store app. Once the Store app opens:
+  1. Use the search bar to look for any desired application. 
+  1. Select from one of curated categories to find essential apps or apps made specifically for HoloLens.
```
Comment (JohanFreelancer9) [inline devices/hololens/holographic-store-apps.md]:
```suggestion
  1. Select essential apps or apps made specifically for HoloLens from one of the curated categories.
```

File: devices/hololens/holographic-store-apps.md [hunk 7]
```diff
@@ -33,20 +33,26 @@ Open the Microsoft Store from the **Start** menu. Then browse for apps and games
...
+  1. On the top right of the Store app select the **"..."** button and select **My Library**, to view any perviously purchased apps.
```
Comment (JohanFreelancer9) [inline devices/hololens/holographic-store-apps.md]:
```suggestion
  1. On the top right of the Store app, select the **"..."** button and select **My Library** to view any previously purchased apps.
```

File: devices/hololens/holographic-store-apps.md [hunk 8]
```diff
@@ -33,20 +33,26 @@ Open the Microsoft Store from the **Start** menu. Then browse for apps and games
...
+1. Select **Get** or **Install** on the application's page (a purchase may be required).
 
 ## Uninstall apps
 
 There are two ways to uninstall applications.  You can uninstall applications through the Microsoft Store or Start menu.
 
 ### Uninstall from the Start menu
 
-On the **Start** menu or in the **All apps** list, gaze at the app. Tap and hold until the menu appears, then select **Uninstall**.
+On the **Start** menu or in the **All apps** list, browse to the app. Air Tap and hold until the menu appears, then select **Uninstall**.
```
Comment (JohanFreelancer9) [inline devices/hololens/holographic-store-apps.md]:
```suggestion
On the **Start** menu or in the **All apps** list, browse to the app. Air tap and hold until the menu appears, then select **Uninstall**.
```

File: devices/hololens/holographic-store-apps.md [hunk 9]
```diff
@@ -33,20 +33,26 @@ Open the Microsoft Store from the **Start** menu. Then browse for apps and games
 
 ## Install apps
 
-To download apps, you'll need to be signed in with a Microsoft account. To buy them, you'll need a payment method associated with the Microsoft account you use on your HoloLens. To set up a payment method, go to [account.microsoft.com](https://account.microsoft.com/) and select **Payment & billing** > **Payment options** > **Add a payment option**.
+To download apps, you'll need to be signed in with a Microsoft account. Some apps are free and can be downloaded right away. Apps that require a purchase require you to be signed into the Store with your Microsoft account and have a valid payment method.
+>[!NOTE]
+>The account you can use on Microsoft Store does not have to be the same as the account you are signed in with. If you are using a Work or School account on your HoloLens then you'll need to sign in with your personal account in the Store App to make a purchase.
```
Comment (JohanFreelancer9) [inline devices/hololens/holographic-store-apps.md]:
```suggestion
> The account you use on Microsoft Store does not have to be the same as the account you are signed in with. If you are using a Work or School account on your HoloLens then you'll need to sign in with your personal account in the Store App to make a purchase.
```

**Comment by scooley** [comment]
@evmill , @JohanFreelancer9  - have all of the copy editing updates happened?  I'd like this to merge before it gets too stake.

File: devices/hololens/holographic-store-apps.md [hunk 10]
```diff
@@ -33,20 +33,26 @@ Open the Microsoft Store from the **Start** menu. Then browse for apps and games
...
+1. Select **Get** or **Install** on the application's page (a purchase may be required).
 
 ## Uninstall apps
 
 There are two ways to uninstall applications.  You can uninstall applications through the Microsoft Store or Start menu.
 
 ### Uninstall from the Start menu
 
-On the **Start** menu or in the **All apps** list, gaze at the app. Tap and hold until the menu appears, then select **Uninstall**.
+On the **Start** menu or in the **All apps** list, browse to the app. Air Tap and hold until the menu appears, then select **Uninstall**.
```
Comment (illfated) [inline devices/hololens/holographic-store-apps.md]: Capitalization: "air tap" is a common expression and not a proper feature name (hence no need for uppercase T)
```suggestion
On the **Start** menu or in the **All apps** list, browse to the app. Air tap and hold until the menu appears, then select **Uninstall**.
```

**Comment by illfated** [comment]
#sign-off

File: devices/hololens/holographic-store-apps.md [hunk 11]
```diff
@@ -33,20 +33,26 @@ Open the Microsoft Store from the **Start** menu. Then browse for apps and games
...
-2. Select the Store app and then tap to place this tile into your world.
-3. Once the Store app opens, use the search bar to look for any desired application.
-4. Select **Get** or **Install** on the application's page (a purchase may be required).
+To set up a payment method, go to [account.microsoft.com](https://account.microsoft.com/) and select **Payment & billing** > **Payment options** > **Add a payment option**.
+
+1. To open the [**Start** menu](holographic-home.md), perform a [Start gesture](https://docs.microsoft.com/hololens/hololens2-basic-usage#start-gesture) or [bloom](hololens1-basic-usage.md) gesture on HoloLens 1.
+1. Select the Store app. Once the Store app opens:
+  1. Use the search bar to look for any desired applications. 
+  1. Select essential apps or apps made specifically for HoloLens from one of the curated categories.
+  1. On the top right of the Store app, select the **"..."** button and then select **My Library** to view any previously purchased apps.
```
Comment (Teresa-Motiv) [inline devices/hololens/holographic-store-apps.md]: Pls remove the quote marks from the button (unless they're actually in the UI.

Comment (illfated) [reply devices/hololens/holographic-store-apps.md]: Good point. I did not think about the UI aspect.

**Comment by e0i** [comment]
@scooley Ready to merge via @Teresa-Motiv

**Comment by illfated** [comment]
@e0i : Looking at the approval ticks and seeing that the tick color is black (not green) for all of us who have submitted our approval, I assume that none of those have rights to merge this PR. Then again, I also presume you will sort that out internally wihtin the MS Docs team. (The GitHub page itself shows a tick comment, when pointing to one of the approval ticks between the comments, saying "Only reviews by reviewers with write access count toward mergeability".)

**Comment by e0i** [comment]
@Dansimp The PR that has been approved by HoloLens writer, Sarah is ready to be merged. Thank you.

**Comment by Teresa-Motiv** [comment]
@Dansimp can you merge this PR?

File: devices/hololens/holographic-store-apps.md (hunk 11 above)
Comment (illfated) [reply devices/hololens/holographic-store-apps.md:46]: @evmill : Please edit this part as suggested by @Teresa-Motiv , or commit my suggestion below:
```suggestion
  1. On the top right of the Store app, select the **...** button and then select **My Library** to view any previously purchased apps.
```

**Comment by illfated** [comment]
Please note the remaining quotation marks around the **...** button comment. I have asked the PR author to make that requested change.

**Comment by e0i** [comment]
@denisebmsft @Dansimp Can you merge this PR that has been approved by the authors? Thank you.

**Pull request merged**